docs/source/tagset/elemento-contrib-group.rst

```diff
@@ -13,7 +13,7 @@ Ocorre:
 
 Contém o grupo de elementos relativos à contribuição na elaboração do artigo. Os contribuintes mais frequentes são os autores pessoais, instituições e grupos de pesquisa.
 
-O elemento pode ou não conter a afiliação, sendo obrigatória quando o atributo tipo de contribuição (@contrib-type) for ``author``, sejam estes autores institucionais ou não.
+O elemento pode ou não conter a afiliação, sendo obrigatória quando o atributo tipo de contribuição (``@contrib-type``) for ``author``, sejam estes autores institucionais ou não.
```

Comment:
@gustavofonseca Ok, por mim sem problemas em retirar todo o parágrafo:

*"O elemento pode ou não conter a afiliação, sendo obrigatória quando o atributo tipo de contribuição (``@contrib-type``) for ``author``, sejam estes autores institucionais ou não."*